router/build.gradle

```diff
@@ -21,6 +21,9 @@ android {
     targetCompatibility 1.8
     sourceCompatibility 1.8
   }
+  kotlinOptions {
+    jvmTarget = "1.8"
+  }
```

Comment:
`NavigatorProvider` has `plusAssign` operator which requires this. Instead of this operator we can use `addNavigator` as well. Should I remove this?

```kotlin
navigationController.navigatorProvider.addNavigator(viewNavigator)
```

Reply:
I think we can use the `addNavigator` method for now.